evap/evaluation/templates/base.html

```diff
@@ -42,30 +42,44 @@
             {% block breadcrumb_bar %}
             {% endblock %}
         </div>
-
+        
```

Comment:
You resolved the comment but didn't change the code here: You added spaces here, which match the indentation level, but are now also trailing spaces on this line.

For consistency within the file, please remove these spaces again.